aether-ui/aether/ui/assets/apps/pipeline/components/DeleteModal.jsx

```diff
@@ -132,13 +150,14 @@ class DeleteModal extends Component {
             }}
           />
           <label for='check3'>
-            <FormattedMessage
-              id='delete.modal.all.data-0'
-              defaultMessage='Data {emphasis} this {objectType} (Entities)'
-              values={{
-                ...messages,
-                emphasis: <b>created by</b>
-              }}
+            <FormattedHTMLMessage
+              {...
+              { ...MESSAGES.entities,
+                values: {
+                  obj: formatMessage(MESSAGES.objectType, { objType: this.props.objectType })
+                }
+              }
+              }
```

Comment:
Same comment 
```javascript
<FormattedHTMLMessage
  id='modal.delete.entities.text'
  defaultMessage='Data <b>created by</b> this {obj} (Entities)'
  values={{ obj: formatMessage(MESSAGES[this.props.objectType]) }}
/>
```

Reply:
Decided to keep all template related messages together

Reply:
in that case (to reduce complexity)...

```
const { formatMessage, formatHTMLMessage } = this.props.intl

// ...

{ formatHTMLMessage(MESSAGE.entities, { obj: formatMessage(MESSAGES[this.props.objectType]) }) }
```

Reply:
Which is what is implemented. using `FormattedHTMLMessage` tag instead